_posts/2019-03-08-code-quality-inspection-for-clojure-with-sonarqube.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Code quality inspection for Clojure using SonarQube  
+author: hjhamala
+excerpt: Clojure has good libraries for code quality analyzing and vulnerabilities detection. I introduce the support for Kibit, ancient-clj, lein-nvd and Cloverage in existing Clojure SonarQube plugin. 
+tags:
+- SonarQube
+- Clojure
+- ClojureScript
+- Code quality
+- Vulnerabilities detection
+- Continuous inspection
+---
+Building good and secure code has never been easy. Programmers make mistakes during development. Programs use external libraries which may have  bugs and vulnerabilities. New vulnerabilities are found which forces updates to existing programs.
```

Comment:
sanoisin: ...may contain bugs...

Ehkä myös jotain:
New vulnerabilities are revealed constantly which enforces patching existing programs.

Reply:
I think "forces" is more correct than "enforces" here.